linux/tools.Dockerfile

```diff
@@ -5,16 +5,16 @@
 
 # To build yourself locally, override this location with a local image tag. See README.md for more detail
 
-ARG IMAGE_LOCATION=cdpxlinux.azurecr.io/artifact/b787066e-c88f-4e20-ae65-e42a858c42ca/official/azure/cloudshell:1.0.20200727.1.base.master.40d4519a
+ARG IMAGE_LOCATION=cdpxlinux.azurecr.io/artifact/b787066e-c88f-4e20-ae65-e42a858c42ca/buddy/azure/cloudshell:1.0.20201009.1.base.add-cbld.5113b5b1
```

Comment:
Do we want to load from this buddy version?

Reply:
Had to merge to master to kick off a new official build. Have now updated this to point to that build. A bit of a messy process...